cfpb/cfgov-refresh
# Adding Samuel Schumach to the press resources page

Adding Samuel Schumach to the press resources page


## Changes

- Modified `cfgov/jinja2/v1/newsroom/press-resources/index.html` to add `Samuel Schumach` to the press resources page.


## Testing

- Visit `http://localhost:8000/about-us/newsroom/press-resources/` 
- Verify that the information and links are correct for `Samuel Schumach`.

## Screenshots

<img width="975" alt="screen shot 2017-03-30 at 11 47 59 am" src="https://cloud.githubusercontent.com/assets/1696212/24513550/7e8a53ac-153f-11e7-8d88-bcc436a6750a.png">

## Notes

-

## Todos

-

## Checklist
